examples/jenkins/pipeline/samplepipeline.json

```diff
@@ -2,12 +2,12 @@
   "kind": "Template",
   "apiVersion": "v1",
   "metadata": {
-    "name": "ruby-helloworld-sample",
+    "name": "sample-pipeline",
     "creationTimestamp": null,
     "annotations": {
-      "description": "This example shows how to create a simple ruby application in openshift origin v3",
+      "description": "This example uses a pipeline to build and deploy a simple ruby application in openshift origin v3",
```

Comment:
"This example showcases the new Jenkins Pipeline integration into OpenShift to perform continuous integration and deployment right from the platform.  Within the sample application is a Jenkinsfile - a definition of a multi-stage CI process that leverages the underlying OpenShift platform for dynamic and scalable builds.  OpenShift integrates the status of your pipeline builds into the web console allowing you to see your entire application lifecycle.  For more information, see <LINK>"